app/controllers/orders_controller.rb

```diff
@@ -21,4 +31,10 @@ def find_order
       flash[:notice] = "Please login appropriately to view that order."
       redirect_to login_path
   end
+
+  def create_item_orders
+    @cart.contents.each do |item, qty|
```

Comment:
we can talk about where this logic should ultimately be, here or in another controller or model